### PR TITLE
create basic and compatability test for nixio_fr

### PR DESCRIPTION
**Pull request by hkchekc**
in Reference to Issue #575 , some basic test for nixio_fr and compatability test of nixio and nixio_fr is added.

### CODE REVIEW

**Comment by JuliaSprenger** [comment]
Hi @hkchekc! Thanks for adding some more tests!
Currently the new tests are failing because of an incomplete path to the test files. You can get the correct path by using `self.get_filename_path(filename)`.

**Comment by JuliaSprenger** [comment]
And you need to inherit from `BaseTestIO` to have this function available. You can compare to https://github.com/NeuralEnsemble/python-neo/blob/master/neo/test/iotest/test_blackrockio.py as an example implementation.

**Comment by hkchekc** [comment]
One test will fail now due to a bug which will be fixed in the PR #581

File: neo/test/iotest/test_nixio_fr.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+class TestNixfr(BaseTestIO, unittest.TestCase, ):
+    ioclass = NixIOfr
+
+    files_to_test = ['nixio_fr.nix']
+
+    files_to_download = ['nixio_fr.nix']
+
+    def setUp(self):
+        super(TestNixfr, self).setUp()
+        self.testfilename = 'nixio_fr.nix'
```
Comment (JuliaSprenger) [inline neo/test/iotest/test_nixio_fr.py]: this line needs to be changed to get the correct file path:
```suggestion
        self.testfilename = 'self.get_filename_path('nixio_fr.nix')
```

**Comment by JuliaSprenger** [comment]
@hkchekc: You will also need to adjust the conversion of nix attributes to regular python objects, because nix objects are not supported by neo eg. as annotations.

**Comment by hkchekc** [comment]
We upload a new nix file for test on Gin.

**Comment by JuliaSprenger** [comment]
I merged the new file and rerun the tests. However, they are still failing somewhere within nix when parsing the header. Do you have an idea how to fix this?

**Comment by JuliaSprenger** [comment]
I merged the new file and rerun the tests. However, they are still failing somewhere within nix when parsing the header. Do you have an idea how to fix this?

**Comment by JuliaSprenger** [comment]
@hkchekc: Thanks for fixing the CI tests.
I still get an error when running the tests locally with numpy 1.15.1 and nixio 1.5.0b2

```python
Failure
  File .../python-neo/neo/rawio/tests/common_rawio_test.py", line 151, in test_read_all
    compliance.read_analogsignals(reader)
  File ".../python-neo/neo/rawio/tests/rawio_compliance.py", line 144, in read_analogsignals
    assert raw_chunk.ndim == 2
AssertionError
```

It seems that `raw_chunk.ndim ==1` in at least one case here. Do you have an idea why this might be the case?

**Comment by hkchekc** [comment]
@JuliaSprenger  I try to use numpy 1.15.1 and nixio 1.5.0b2 to run the test_nixio_fr.py and test_nixrawio.py for both 'nixrawio-1.5.nix' and ' nixio_fr.nix' and could not reproduce the error. 

Can you tell me more details?

**Comment by JuliaSprenger** [comment]
Hi @hkchekc,
I had a closer look and think that the error was somehow related to an improper setup of my environment. If I create a new environment everything works smoothly. Sorry for the confusion.
I will merge this now.

**Comment by hkchekc** [comment]
Thanks for all the checkings! @JuliaSprenger

**Pull request merged**